app/src/main.ts

```diff
@@ -69,6 +69,8 @@ if (!appArgs.userAgentHonest) {
   }
 }
 
+app.setAppUserModelId(app.getName());
```

Comment:
Two things:

1. Pls wrap in an `if (isWindows())` . We have such a helper function somewhere.
2. Comment, pls, and link to https://www.electronjs.org/docs/latest/api/app#appsetappusermodelidid-windows